README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
+ sudo apt install gradle
+- Install Canberra
+ Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️What is Gradle
+
+Gradle is a automated build system that makes working with large projects easier.
+It streams line the build process by skipping re-execution and allows developers to preform tasks suing one line of code in the command line.
+
+
+
+◾️How does Gradle Work?
+
+▪️Project 
+- A project is a depiction of what needs to be built and how we get an artifact at the end of a build.
+
+settings.gradle - Where projects are registered.
+build.gradle - constructs commands that can be used to execute tasks.
+
+
+▪️Task
+-A task is used to execute a action during the build process. For example, ./gradlew compileJava will compile the main class in the build. 
+
+
+
+▪️3 Build Phases
+
+The Initialization Phase 
+- In this phase Gradle tires to establish what projects are in the build process.
+Gradle will look to the file setting.gradle to identify the project or projects. 
+
+The Configuration Phase
+- In this phase Gradle locates and executes build scripts for each project. This would be in our case the build.gradle file. Understand that when the build script is executed the tasks inside the build script are not. From this Gradle uses an algorithm called DAG that creates a sort of map to what dependents are needed for each task. In short this allows us to use gradle commands.
+
+The Execution Phase
+- In this phase Gradle locates what tasks are dependent on each other, and is able to actually start the build process. This is where we are able to execute tasks in the command line in order to clean, compile, run, and use other commands to build the project. 
+
+
+
+
+◾️Possible Errors and Solutions During Build
+Error
+-Failed to load module "canberra-gtk-module"
+Solution
+-Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️Key Commands
+./gradlew tasks --all | Displays the full list of commands that can be excecuted
+./gradlew run | Runs the program
+./gradlew compileJava | Compiles the main java source
+./gradlew clean | Deletes the build Directory
+./gradlew classes | Assembles the main classes
+./gradlew build | Assembles and tests the project.
+./gradlew testClasses | Assembles the test classes
+./gradlew checkstyle | generates checkstyle reports
+./gradlew test | Runs the unit tests.
+./gradlew compileTestJava | Compiles the Unit Tests
+./gradlew check | Runs all checks
```

Comment:
The use of the pipe symbol  `|`  to indicate a comment is potentially confusing.

Consider using different formatting to distinguish `build commands` from _descriptions_.

This also seems to be a reverse-lookup.  The key is the command and then you get its meaning.  Wouldn't someone trying to build the product actually be searching for the command associated with what they are trying to accomplish rather than seeing a command and trying to figure out what it does?